plugin.video.catchuptvandmore/resources/lib/openvpn.py

```diff
@@ -0,0 +1,211 @@
+# /*
+# *
+# * OpenVPN for Kodi.
```

Comment:
Inorder to merge this plugin you need to remove any code that circumvents geo restrictions. VPN's are not allowed.